From 9fe47e98a7f3b542413f3793912b291599f05860 Mon Sep 17 00:00:00 2001
From: Thomas Devoogdt <thomas.devoogdt@barco.com>
Date: Tue, 17 Jan 2023 00:55:33 -0800
Subject: [PATCH] REGRESSION(258347@main): Fix webkit.h public header

https://bugs.webkit.org/show_bug.cgi?id=250701

Reviewed by Carlos Garcia Campos.

Since commit 654646fe6187abcf9ced6a3ace80eaf04754fd39, the webkit.h file is generated
and does only define __WEBKIT_H_INSIDE__. So replace the only remaining __WEBKIT2_H_INSIDE__
defines to avoid any "Only <webkit2/webkit2.h> can be included directly." error.

Signed-off-by: Thomas Devoogdt <thomas.devoogdt@barco.com>

Canonical link: https://commits.webkit.org/258974@main
---
 Source/WebKit/Scripts/glib/generate-api-header.py        | 4 ++--
 Source/WebKit/UIProcess/API/gtk/WebKitEnumTypesGtk3.h.in | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/Source/WebKit/Scripts/glib/generate-api-header.py b/Source/WebKit/Scripts/glib/generate-api-header.py
index 6eb326b8764f..6977b51914e4 100644
--- a/Source/WebKit/Scripts/glib/generate-api-header.py
+++ b/Source/WebKit/Scripts/glib/generate-api-header.py
@@ -30,7 +30,7 @@
     "gtk4": '''#if !defined(__WEBKIT_H_INSIDE__) && !defined(BUILDING_WEBKIT)
 #error \"Only <webkit/webkit.h> can be included directly.\"
 #endif''',
-    "gtk": '''#if !defined(__WEBKIT2_H_INSIDE__) && !defined(BUILDING_WEBKIT)
+    "gtk": '''#if !defined(__WEBKIT_H_INSIDE__) && !defined(BUILDING_WEBKIT)
 #error \"Only <webkit2/webkit2.h> can be included directly.\"
 #endif''',
     "wpe": '''#if !defined(__WEBKIT_H_INSIDE__) && !defined(BUILDING_WEBKIT)
@@ -54,7 +54,7 @@
     "gtk4": '''#if !defined(__WEBKIT_H_INSIDE__) && !defined(__WEBKIT_WEB_EXTENSION_H_INSIDE__) && !defined(BUILDING_WEBKIT)
 #error \"Only <webkit/webkit.h> can be included directly.\"
 #endif''',
-    "gtk": '''#if !defined(__WEBKIT2_H_INSIDE__) && !defined(__WEBKIT_WEB_EXTENSION_H_INSIDE__) && !defined(BUILDING_WEBKIT)
+    "gtk": '''#if !defined(__WEBKIT_H_INSIDE__) && !defined(__WEBKIT_WEB_EXTENSION_H_INSIDE__) && !defined(BUILDING_WEBKIT)
 #error \"Only <webkit2/webkit2.h> can be included directly.\"
 #endif''',
     "wpe": '''#if !defined(__WEBKIT_H_INSIDE__) && !defined(__WEBKIT_WEB_EXTENSION_H_INSIDE__) && !defined(BUILDING_WEBKIT)
diff --git a/Source/WebKit/UIProcess/API/gtk/WebKitEnumTypesGtk3.h.in b/Source/WebKit/UIProcess/API/gtk/WebKitEnumTypesGtk3.h.in
index 52cda32496bb..493b8e9a1e5e 100644
--- a/Source/WebKit/UIProcess/API/gtk/WebKitEnumTypesGtk3.h.in
+++ b/Source/WebKit/UIProcess/API/gtk/WebKitEnumTypesGtk3.h.in
@@ -18,7 +18,7 @@
  * Boston, MA 02110-1301, USA.
  */
 
-#if !defined(__WEBKIT2_H_INSIDE__) && !defined(BUILDING_WEBKIT)
+#if !defined(__WEBKIT_H_INSIDE__) && !defined(BUILDING_WEBKIT)
 #error "Only <webkit2/webkit2.h> can be included directly."
 #endif
 
From 8f11ad0813c37f4bd8ee43ecac733892a57e2a03 Mon Sep 17 00:00:00 2001
From: Thomas Devoogdt <thomas.devoogdt@barco.com>
Date: Tue, 17 Jan 2023 05:45:34 -0800
Subject: [PATCH] REGRESSION(258347@main): Fix webkit.h public header part 2

https://bugs.webkit.org/show_bug.cgi?id=250701

Reviewed by Carlos Garcia Campos.

In commit 9fe47e98a7f3b542413f3793912b291599f05860, I've replaced all
instances of __WEBKIT2_H_INSIDE__ with __WEBKIT_H_INSIDE__ but overlooked
this one. So fix that.

Signed-off-by: Thomas Devoogdt <thomas.devoogdt@barco.com>

Canonical link: https://commits.webkit.org/258976@main
---
 Source/WebKit/UIProcess/API/gtk/WebKitVersion.h.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Source/WebKit/UIProcess/API/gtk/WebKitVersion.h.in b/Source/WebKit/UIProcess/API/gtk/WebKitVersion.h.in
index 0704d8016c45..f141a20aa65d 100644
--- a/Source/WebKit/UIProcess/API/gtk/WebKitVersion.h.in
+++ b/Source/WebKit/UIProcess/API/gtk/WebKitVersion.h.in
@@ -17,7 +17,7 @@
  * Boston, MA 02110-1301, USA.
  */
 
-#if !defined(__WEBKIT@WEBKITGTK_API_INFIX@_H_INSIDE__) && !defined(__WEBKIT_WEB_EXTENSION_H_INSIDE__) && !defined(BUILDING_WEBKIT)
+#if !defined(__WEBKIT_H_INSIDE__) && !defined(__WEBKIT_WEB_EXTENSION_H_INSIDE__) && !defined(BUILDING_WEBKIT)
 #error "Only <webkit@WEBKITGTK_API_INFIX@/webkit@WEBKITGTK_API_INFIX@.h> can be included directly."
 #endif
 
